tools/bazel.rc

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 The Cartographer Authors
```

Comment:
Is it important that this is in tools? We have a cmake subdirectory. Should we rather have this file in a bazel subdirectory?

Reply:
As discussed this one needs to stay but others have gone into the subfolder.

Reply:
https://github.com/bazelbuild/bazel/issues/4502